pkg/client/cache/listers.go

```diff
@@ -39,7 +39,7 @@ import (
 // l := StoreToPodLister{s}
 // l.List()
 type StoreToPodLister struct {
-	Store
+	Store Indexer
```

Comment:
Basically Indexer interface contains Store inside, so Indexer is totally enough.